modules/reference/pages/home/home-notification-messages.adoc

```diff
@@ -1,84 +1,42 @@
 [[ref.webui.overview.notifications]]
 = Notification Messages
 
-The menu:Main Menu[Home > Notification Messages] page allows you to manage your notification messages of the {productname} server.
+The menu:Home[Notification Messages] section shows all current messages produced by {productname}.
+By default, messages will remain current for thirty days.
+After this period, messages are deleted whether or not they are marked as read.
 
-image::home_notification_messages.png[scaledwidth=80%]
+To see unread messages, navigate to the [guimenu]``Unread Messages`` tab.
+For see all messages, navigate to the [guimenu]``All Messages`` tab.
+
+Click btn:[Refresh] to update the list.
+
+Perform bulk actions by checking messages in the list.
+Click btn:[Delete selected messages] to bulk delete messages.
+Click btn:[Mark selected as read] to bulk read messages.
+
+[[message-severity-status]]
+.Notification Message Severity Statuses
+[cols="1,1,1", options="header"]
+|===
+| Icon               | Description   | Example
+| icon:info-circle[role="none"] | Information | Client onboarding has failed.
+| icon:exclamation-triangle[role="orange"] | Warning | Channel synchronization has completed.
+| icon:times-circle[role="red"] | Error | Channel synchronization has failed.
+|===
+
+
+// needs to be moved to Admin or something, I think. LKB 2020-02-20
 
 The default lifetime of notification messages is 30 days, after which messages are deleted from the database, regardless of read status.
-You can configure the amount of time in [path]``/etc/rhn/rhn.conf``:
+To change this value, add or edit this line in [path]``/etc/rhn/rhn.conf``:
 
 ----
-# Configure the lifetime of notification messages in days
 java.notifications_lifetime = 30
 ----
 
 All notification types are enabled by default.
-To disable a notification type, open the [path]``/etc/rhn/rhn.conf`` configuration file and edit the [systemitem]``java.notifications_type_disabled`` parameter:
+To disable a notification type, add or edit this line in [path]``/etc/rhn/rhn.conf``:
 
 ----
-# Configure the disablement of notification messages by type - example disabling all notification types
 java.notifications_type_disabled = OnboardingFailed,ChannelSyncFailed,ChannelSyncFinished
 ----
```

Comment:
Wondering why you removed the rest.  But ok.

Reply:
Agreed, is there a reason we are removing the legend? It probably needs some cleanup but this is a large chunk.

Reply:
The legend is now in the table.